railties/lib/rails/tasks/restart.rake

```diff
@@ -1,4 +1,5 @@
 desc "Restart app by touching tmp/restart.txt"
 task :restart do
+  Dir.mkdir('tmp') unless File.directory('tmp')
```

Comment:
@chancancode why `mkdir_p` when there are not parent directories to create? Am I missing something?